runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/HeroSection/HeroBackground/HeroBackground.module.scss

```diff
@@ -155,9 +155,28 @@ $ARROWS_HEIGHT: 176px;
     }
 }
 
+
 @media (min-width: $breakpoint-xxl) {
     .heroArrows {
         background-position: center;
     }
 }
 
+@media (min-height: 700px) {
```

Comment:
stick to defined breakpoints in breakpoint css file

Reply:
media queries are designed to work with the width property. Try to solve it the other way